.github/workflows/ci-pipeline-arm64.yml

```diff
@@ -98,7 +98,8 @@ jobs:
         conf: ['', '--enable-abi-compat', '--enable-deprecated --enable-debug=full',
                '--enable-dpdk-zero-copy --disable-static-applications',
                '--disable-host-optimization', '--disable-host-optimization --enable-abi-compat',
-               '--without-openssl --without-pcap']
+               '--without-openssl --without-pcap', 'CFLAGS="-march=armv8.2-a"',
+               'CFLAGS="-march=armv8-a+lse"']
```

Comment:
It's probably a good idea to add also `-O2` flag, since it previously revealed an implementation bug.

Reply:
I have made this change in v4. Thanks!